docs/releasing-files.md

```diff
@@ -41,6 +41,20 @@ It may be possible for exceptions to be made if they can be justified as being b
 
 **If you are unsure about anything, please email us: <disclosurecontrol@opensafely.org>.**
 
+## Publication of your study results (released from the Level 4 environment) and GitHub repository
+
+### Study results
+Once the Level 4 outputs have been released (as per the above process), you can share the data **in confidence** with wider members of the research team, including external collaborators. For example, the study author may copy the outputs from the GitHub repository to a document being prepared for publication, or to seek feedback and contribution from other colleagues to inform the final paper or report. 
+
+However, **for publication (e.g. MedRxiv), or circulation of the findings outside of the immediate research team or collaborators** (e.g. presentations given to interested groups), the final paper/report/presentation (etc.), including a brief lay summary of the important conclusions, **must first be emailed to <amir.mehrkar@phc.ox.ac.uk> who will seek final approval for release from NHS England.** The purpose of this approval step is to check that the paper/report aligns with the stated application purpose.
+
+### Study GitHub repository
+
+Similarly, **the study repository must be kept private until project completion and NHS England approval has been obtained**; however, the OpenSAFELY Oversight Board has recommended that all study GitHub repositories (to include the study definition code, codelists and released results) be make public 12 months after any code has been executed against the underlying patient data.
```

Comment:
This isn't quite right, if the repository doesn't contain any outputs - right?

Reply:
needs to marry up with point 7 on latest Data Access Agreement want to send out too https://docs.google.com/document/d/1Kf8vL1Gy8t_AJgTTOcPtwl1vlBNQSCu855Bd4JUoL9I/edit so if we change this (which is fine), need to update the DAA too.

Reply:
Please HOLD OFF on this commit

Reply:
We may have another solution that need to work up and discuss with Ben and Seb Tuesday